v3/src/components/graph/adornments/univariate-measures/normal-curve/normal-curve-adornment-component.tsx

```diff
@@ -367,6 +366,10 @@ export const NormalCurveAdornmentComponent = observer(
       }
     }, [])
 
+    if (graphModel.plotType === "binnedDotPlot") {
```

Comment:
Now that we've got plot models, I think we should move away from explicit plot type tests in favor of delegating responsibility to the plot models. Could this be `graphModel.plot.isBinned`?